File: src/main/java/com/tobe/healthy/push/repository/MemberTokenRepository.java
package com.tobe.healthy.push.repository;

import java.util.Optional;

import org.springframework.data.jpa.repository.JpaRepository;

import com.tobe.healthy.push.domain.entity.MemberToken;

public interface MemberTokenRepository extends JpaRepository<MemberToken, Long> {

	Optional<MemberToken> findByMemberId(Long memberId);
}

File: src/main/kotlin/com/tobe/healthy/push/repository/MemberTokenRepository.kt
package com.tobe.healthy.push.repository

import com.tobe.healthy.push.domain.entity.MemberToken
import org.springframework.data.jpa.repository.JpaRepository

interface MemberTokenRepository : JpaRepository<MemberToken, Long> {
    fun findByMemberId(memberId: Long): MemberToken?
}